src/components/Counter.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sourceRoot":"","sources":["Counter.scss","../theme/typography.scss","../theme/colors.scss"],"names":[],"mappings":"AAGA;ECIE,aAPgB;EAQhB;EACA;;;ADFF;ECAE,aAPgB;EAQhB;EACA;EDCA;EACA,kBEXK;EFYL;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;AAEA;EACE,kBEtBW;;AFyBb;EACE,kBExBI","file":"Counter.css"}
```

Comment:
.gitignore

Reply:
should just be deleted can that file imported to App.scss.  also, soon to be deleted.